_posts/2020-03-27-status-update.md

```diff
@@ -0,0 +1,67 @@
+Here's a brief overview of what we've been up to:
+
+## COVID-19 
+
+Altough this situation affects all of us, we do our best to continue the good work. We chose to stay home in order to protect the community and stop the spread of the virus. We all miss the office and the chats about the weather next to the coffee machine, but here we have some friends, who are happy that we work remote: 
+
+
+Read [Puppet's blog: COVID-19](https://puppet.com/blog/covid-19-a-note-for-our-community/) for the official statement.
+
+## Updates to Puppet Modules
+
+Over the last week we have released:
+
+- [puppetlabs/translate](https://forge.puppet.com/puppetlabs/translate) (v2.2.0)
+- [puppetlabs/firewall](https://forge.puppet.com/puppetlabs/firewall) (v2.3.0)
+
+## Community
+
+Puppet Community was always a part of our team and we always find new interesting things to add in our modules. Thank you, guys ! Here are a few highlights of this week :
+
+- [puppetlabs-postgresql#1156](https://github.com/puppetlabs/puppetlabs-postgresql/pull/1156) by [veninga](https://github.com/veninga)
+- [puppetlabs-firewall#908](https://github.com/puppetlabs/puppetlabs-firewall/pull/908) by [jbro](https://github.com/jbro)
+- [puppetlabs-postgresql#1158](https://github.com/puppetlabs/puppetlabs-postgresql/pull/1158) by [XnS](https://github.com/XnS)
+
+We look forward to see your new contributions to Puppet modules.
+
+## Customer Escalations
+
+This week, the team worked at Panos module. (check out this [PR](https://github.com/puppetlabs/puppetlabs-panos/pull/119) for more details). Our purpose is to unblock testing and progress development.
+
+## DSC Rework
+
+We've started work on re-architecting and revitalizing our DSC builder (check out [IAC-41](https://tickets.puppetlabs.com/browse/IAC-41) for some details!) based on prototype work by [James Pogran](https://github.com/jpogran). In short, this project is aiming to provide a PowerShell module for Puppetizing DSC resources, complete with intellisense, help, and guardrails.
+A bit further out and we'll have a new namespace on the forge - dsc - where we'll publish these automatically generated modules as 1:1 matches for the PowerShell modules with DSC resources on the [PowerShellGallery](https://www.powershellgallery.com/packages)!
+We're working hard right now to pull the prototype across the last couple of miles to get an MVP for testing - if you're interested in beta testing this project with us, please reach out on [slack in #windows](https://slack.puppet.com/) or via [email](ia-content@puppet.com)
+
+## Litmus Progress
+
+On [Litmus](https://github.com/puppetlabs/puppet_litmus), the team had a productive week. We managed to fix some bugs that you can check it [here](https://github.com/puppetlabs/puppet_litmus/pull/271) and [here](https://github.com/puppetlabs/puppet_litmus/pull/272). Also, now we are able to capture more informations for 
+[honeycomb](https://github.com/puppetlabs/puppet_litmus/pull/264). 
+
```

Comment:
Maybe also add this line about our progress with the core module Litmus ports:
```suggestion
We only have two of our core modules left to convert to Litmus. Work has begun on [puppetlabs-reboot](https://forge.puppet.com/puppetlabs/reboot), with [puppetlabs-ibm_installation_manager](https://forge.puppet.com/puppetlabs/ibm_installation_manager) being the last module on the list! If you’re interesting in converting a module to Litmus, feel free to get in touch with us during [Office Hours](https://puppet.com/community/office-hours/)
```

Reply:
@daianamezdrea Could you please add the following modules too
Work has begun on [puppetlabs-websphere_application_server](https://forge.puppet.com/puppetlabs/websphere_application_server) and [puppetlabs-satellite_pe_tools](https://forge.puppet.com/puppetlabs/satellite_pe_tools)

Reply:
Sure, @sheenaajay , thank you!